src/pages/publication/NewPublication.tsx

```diff
@@ -81,7 +76,7 @@ const NewPublication: FC = () => {
               />
             </StyledSelectorWrapper>
             <StyledInfoBox>
```

Comment:
her fikk jeg ikke til å arve fra mui-Card pga lokal stiling ikke ble tatt hensyn til

Reply:
works læll

Reply:
måtte legge inn ...props for å få det til å fungere